### ISSUE TITLE
Helicopter crashes leave you completely unharmed

### ISSUE THREAD

**Issue by ERRORCODE509** [CleverRaven/Cataclysm-DDA#57429]
### Describe the bug

Crashing a helicopter (both by hitting something and by running out of fuel) will liquify your helicopter, but be leave you completely unharmed. It doesn't even cause damage due to the fall, let alone the crash. It seemingly teleports you too the ground without any consequences (other than losing your helicopter).

### Steps to reproduce

1. Make a new game
2. Spawn in a helicopter
3. Fly up to the max height and let go of the controls
4. Crash the helicopter
5. You are unharmed
6. Spawn another and crash it into a building
7. You are still unharmed

### Expected behavior

That helicopter crashes are lethal, or at least damaging.

### Versions and configuration

- OS: Android
    - OS Version: Manufacturer: samsung; Model: SM-N960U; Release: 10; Incremental: N960USQS9FVC5; 
- Game Version: 0.F-3 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions],
]

### Additional context

I've tested this on two recent-ish experimentals as well as the stable build (as shown here) and the big was present in all of them.

**Comment by Night-Pryanik** [comment]
This happens not only with helicopters, but with most other vehicles (for example, when they're falling from roofs).

**Comment by AlbertNewton** [comment]
You should at least take damage comparable to the Helicopter Crash start.w

**Comment by ERRORCODE509** [comment]
> You should at least take damage comparable to the Helicopter Crash start.

Agreed. I do think that the damage you receive in the Helicopter Crash start is way too low, however. A helicopter crash isn't exactly the kind of thing you'd walk away from with just a few cuts and minor damage—depending on the height you crashed from, of course.

**Comment by AlbertNewton** [comment]
Autorotation exists; most helicopters are designed to land survivably with limited damage in case of engine failure.
https://en.wikipedia.org/wiki/Autorotation
"The most common use of autorotation in helicopters is to safely land the aircraft in the event of an engine failure or tail-rotor failure. It is a common emergency procedure taught to helicopter pilots as part of their training."

**Comment by ERRORCODE509** [comment]
> Autorotation exists; most helicopters are designed to land survivably with limited damage in case of engine failure.
> https://en.wikipedia.org/wiki/Autorotation
> "The most common use of autorotation in helicopters is to safely land the aircraft in the event of an engine failure or tail-rotor failure. It is a common emergency procedure taught to helicopter pilots as part of their training."
> 
> Of course, that doesn't account for the destroyed state of the helicopter; maybe the zombies have been smashing it up, maybe you just got lucky to have such light injuries.

Interesting, didn't know that. This could be relevant as well: https://en.m.wikipedia.org/wiki/Helicopter_height%E2%80%93velocity_diagram
